src/pages/community.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Community
+description:
+  This page contains a curated list of interesting Front-Commerce community
+  projects and resources.
+---
+
+<p>{frontMatter.description}</p>
+
+import ContactLink from "@site/src/components/ContactLink";
+
+Please <ContactLink>let us know</ContactLink> if you want to share a project
+here.
+
+# Examples
```

Comment:
After adding the title this can be h2, same for Extensions, and then the other two can be h3

Reply:
I amended the commit with this change.